wpiutil/build.gradle

```diff
@@ -137,7 +138,23 @@ cppHeadersZip {
     }
 }
 
+        // it.component.sources.each {
+        //     it.exportedHeaders {
+        //         srcDirs 'src/main/native/include', 'src/main/native/libuv/include', 'src/main/native/libuv/src'
+        //     }
+        // }
+
```

Comment:
Seems like this section should be deleted instead of commented out?

Reply:
Hmm problably, don't know how that got there.

Reply:
Fixed